jina/flow/base.py

```diff
@@ -364,7 +363,7 @@ def __init__(
         self.k8s_infrastructure_manager = None
         if self.args.infrastructure == InfrastructureType.K8S:
             self.k8s_infrastructure_manager = self._FlowK8sInfraResourcesManager(
-                k8s_namespace=self.args.name,
+                k8s_namespace=self.args.k8s_namespace,
```

Comment:
is `k8s_namespace` a `Flow` args as well? can u change some tests in `k8s_tests` where u can check that the `k8s_namespace` can differ from the flow name? Otherwise I guess the k8s_namespace may be empty

Reply:
added to most of the tests (not all), i see the namespace is correct, not sure how to programmatically check the namespace

Reply:
u can check once the flow is started, using kubernetes client that the namespace u expect exists

Reply:
and with the expected deployments

Reply:
added